src/test/java/uk/gov/hmcts/darts/utilities/DataUtilTest.java

```diff
@@ -0,0 +1,78 @@
+package uk.gov.hmcts.darts.utilities;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import uk.gov.hmcts.darts.model.audio.AddAudioMetadataRequest;
+import uk.gov.hmcts.darts.model.audio.AddAudioMetadataRequestWithStorageGUID;
+
+import java.time.OffsetDateTime;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.UUID;
+
+class DataUtilTest {
+
+
+    private AddAudioMetadataRequest getAddAudioMetadataRequest() {
+        AddAudioMetadataRequest addAudioMetadataRequest = new AddAudioMetadataRequest();
+        addAudioMetadataRequest.setStartedAt(OffsetDateTime.parse("2021-01-01T00:00:00Z"));
+        addAudioMetadataRequest.setEndedAt(OffsetDateTime.parse("2022-01-01T00:00:00Z"));
+        addAudioMetadataRequest.setChannel(1);
+        addAudioMetadataRequest.setTotalChannels(2);
+        addAudioMetadataRequest.setFormat("format");
+        addAudioMetadataRequest.setFilename("filename");
+        addAudioMetadataRequest.setCourthouse("courthouse");
+        addAudioMetadataRequest.setCourtroom("courtroom");
+        addAudioMetadataRequest.setMediaFile("mediaFile");
+        addAudioMetadataRequest.setFileSize(100L);
+        addAudioMetadataRequest.setChecksum("checksum");
+        addAudioMetadataRequest.setCases(List.of("case1", "case2"));
+        return addAudioMetadataRequest;
+    }
+
+    @Test
+    void positiveToMap() {
```

Comment:
Do we need any negative tests here?

Reply:
I have added some null checks to ensure we don't break if the object is missing data (but this "should" be filtered out when we receive the API but its always good from a belts and braces viewpoint to verify)